### PR TITLE
The core.Logger#setLevel method should work like

### PR DESCRIPTION
**Pull request by garydgregory**
The core.Logger#setLevel method should work like Configurator#setLevel #2281

@ppkarwasz 

I'm not sure how to really test this from our 1.2 module but I did install a snapshot locally and tested it with my Apache Commons Logging branch here https://github.com/garydgregory/commons-logging/tree/log4j1-log42-api

The POM change to apache-rat-plugin was required for `mvn clean verify` to pass locally.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -542,6 +542,7 @@
       <plugin>
         <groupId>org.apache.rat</groupId>
         <artifactId>apache-rat-plugin</artifactId>
+        <version>0.16.1</version>
```
Comment (ppkarwasz) [inline pom.xml:545]: We should bump this in `logging-parent` if necessary or even better in the ASF parent: apache/maven-apache-parent#193

**Review by ppkarwasz — COMMENTED** [review]
Can we simply test that setting a new level on a "foo" logger changes the effective level on a "foo.bar" logger?

**Comment by ppkarwasz** [comment]
BTW: you should also add a file in `src/changelog/.2.x.x`.

**Comment by garydgregory** [comment]
I added tests but they fail. I can't tell if the issue is in main or the test itself...

**Pull request merged**